src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksShelf.tsx

```diff
@@ -0,0 +1,98 @@
+import { Flex, Skeleton } from "@artsy/palette"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { FC } from "react"
+import { graphql } from "react-relay"
+import {
+  SuggestedArtworksShelfQuery,
+  SuggestedArtworksShelfQuery$data,
+} from "__generated__/SuggestedArtworksShelfQuery.graphql"
+
+import {
+  ShelfArtworkFragmentContainer,
+  ShelfArtworkPlaceholder,
+} from "Components/Artwork/ShelfArtwork"
+import { SearchCriteriaAttributes } from "Components/SavedSearchAlert/types"
+import { extractNodes } from "Utils/extractNodes"
+
+export const NUMBER_OF_ARTWORKS_TO_SHOW = 5
+
+interface SuggestedArtworksShelfProps {
+  artworksConnection: SuggestedArtworksShelfQuery$data["artworksConnection"]
+}
+
+export const SuggestedArtworksShelf: FC<SuggestedArtworksShelfProps> = ({
+  artworksConnection,
+}) => {
+  const artworks = extractNodes(artworksConnection)
+
+  return (
+    <Flex flexDirection="row" flexWrap="wrap" gap={2} justifyContent="center">
```

Comment:
Why aren't we using an actual `Shelf` component? What this never overflows?

Reply:
Yes, it doesn't overflow - it just displays the first 5 suggested artworks. There is a "See more" button underneath it that opens modal where we see all of the artworks.

Reply:
What does it look like when it wraps? I think these components are variable width, sized based on area; so for instance with some landscapes in here?

Reply:
The idea was to wrap them underneath (centered) but now that you mention it - it sounds better to have them scale instead of wrapping, I will suggest it!

Reply:
That will probably wind up being quite difficult to achieve using the current components. It would probably make sense to remove them completely, as they overflow. Which would be easier to achieve.

Reply:
Removing the wrap keeps the default behaviour where it scales on smaller screens, but I will discuss with design as well and update in the next PR.